app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -27,15 +27,22 @@ def create
       # It's important we store the ID and type of the QB journal entry we just created
       # so that on the next sync, a duplicate is not created.
       @transaction.associate_with_qb_obj(journal_entry)
-
       @transaction.save!
+
+      # Create blank interest transaction. The interest calculator will pick this up and
+      # calculate the value, and sync it to quickbooks.
+      interest_transaction = ::Accounting::Transaction.find_or_initialize_by(transaction_interest_params
```

Comment:
I think this would be clearer as like `transaction_params.except(:amount, ...)`. I was confused at first why there were two _params methods, suggesting two different forms or something...